noxfile.py

```diff
@@ -39,7 +39,7 @@ def lint(session):
     session.run("flake8", "google", "tests")
 
 
-@nox.session(python="3.6")
+@nox.session(python="3.7")
```

Comment:
Do we want to change this?

Reply:
Probably not if we switch to Kokoro in the future